packages/release/release-sysctl.conf

```diff
@@ -2,8 +2,14 @@
 # Maximize console logging level for kernel printk messages
 kernel.printk = 8 4 1 7
 
-# Wait 30 seconds and then reboot
-kernel.panic = 30
+# Wait 10 seconds and then reboot
+kernel.panic = 10
```

Comment:
I believe `kube-bench` complains if this is higher than 10s (based on https://rancher.com/docs/rancher/v2.x/en/security/rancher-2.4/hardening-2.4/#configure-kernel-runtime-parameters perhaps?)

Would a lower setting be OK; for example, 2 seconds or 5 seconds?

Reply:
I expect it would be OK, but I have trouble picturing a scenario where it would make a big difference. The node will still take 10+ seconds to come back up from a reboot, which will usually mean a noticeable service impact.